src/utils/formValidation.ts

```diff
@@ -1,6 +1,40 @@
 export const mustBePresent = (value: any): string | undefined =>
   value ? undefined : 'validation.message.required';
 
+export const mustBeNames = (maxNames: number) => (value: any): string | undefined => {
+  const regexString = `^(?! )([\\p{Script_Extensions=Latin}-]+\\s*){1,${maxNames}}(?<! )$`;
```

Comment:
Why are `\\p` and `\\s` escaped? Shouldn't they be `\p` and `\s`?

Reply:
Also, `lookbehind` doesn't seem to have good support in JS.

ref: https://caniuse.com/js-regexp-lookbehind

Reply:
> Why are `\\p` and `\\s` escaped? Shouldn't they be `\p` and `\s`?

This is a template string with a regex. If `\p` is not escaped as `\\p`, the end result is `p`

> Also, `lookbehind` doesn't seem to have good support in JS.
> 
> ref: https://caniuse.com/js-regexp-lookbehind

Sadly true, the app breaks completely on Safari. I need to take another look at this then